keps/sig-node/213-run-as-group/README.md

```diff
@@ -223,17 +223,141 @@ There are other potentially unresolved discussions in that PR which need a follo
 - https://github.com/kubernetes/website/pull/12297
 - https://github.com/kubernetes/kubernetes/pull/73007
 
+## Test Plan
+For `Alpha`, unit tests and e2e tests were added to test functionality at both
+container and pod level for dockershim.
+
+For `Beta`, tests were added to other CRI's like cri-o, containerd and Docker.
+
+For `GA`, the introduced e2e tests will be promoted to conformance. It was also
+verified that all e2e coverage was proper and CRI's had tests in their respective
+repos testing this feature.
 
 ## Graduation Criteria
 
-- Publish Test Results from Master Branch of Cri-o To http://prow.k8s.io [#72253](https://github.com/kubernetes/kubernetes/issues/72253)
-- Containerd and CRI-O tests included in k/k CI [#72287](https://github.com/kubernetes/kubernetes/issues/72287)
-- Make CRI tests failures as release informing
+Beta
+- RunAsGroup is tested for containerd and CRI-O in cri-tools repo using critest
+  --  [Tests](https://github.com/kubernetes-sigs/cri-tools/blob/16911795a3c33833fa0ec83dac1ade3172f6989e/pkg/validate/security_context_linux.go#L357)
+- critests are executed in cri-tools for all merges as GitHub Action
+  --  [CRI-O](https://github.com/kubernetes-sigs/cri-tools/actions?query=workflow%3A%22critest+CRI-O%22)
+  --  [containerd](https://github.com/kubernetes-sigs/cri-tools/actions?query=workflow%3A%22critest+containerd%22) 
+
+GA
+- assuming no negative user feedback, promote after 1 release at beta.
+- verify test coverage for CRI's
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+This section must be completed when targeting alpha to a release.
+NA, since this targetting Beta to GA.
```

Comment:
I'd like to see this filled out. It will be a long time before most people are to 1.21. This serves as their guide before then.